source/maintenance-guide/maintenance-guide.rst

```diff
@@ -178,6 +178,33 @@ More information can be found at https://www.chirpstack.io/application-server/in
 
 For installation configuration of Chirpstack see: https://www.chirpstack.io/application-server/install/config/
 
+Migrations
+-----------
+
+This project is using TypeORM migrations when changes are applied to the database. This is recommended by TypeORM when the project is used for production.
+
+Generate migrations
+~~~~~~~~~~~~~~~~~~~
+
+If you make some changes in the database you need to generate the migration. In the console you write: "npm run generate-migration <your name of the migration>". Then a migration file 
+will be created in a Migration folder. A timestamp will be added to the name to indicate when the migration has been generated. If no changes are made to the database, nothing will happen.
```

Comment:
Reword the last sentence. Something like "If no changes have been made to the entity classes, no migrations are generated"